models/record/fields.yaml

```diff
@@ -432,11 +432,6 @@ secondaryTabs:
                         label: Description
                         type: textarea
                         size: small
-        blocks:
-            label: Grouped Repeater
-            tab: Repeater
-            type: repeater
-            groups: ~/plugins/winter/test/models/record/blocks.yaml
```

Comment:
Was this intentionally removed?

Reply:
@mjauvin why?

Reply:
Because it's not used enaywhere currently and was causing problems when saving the model.